src/components/DebuggerV2/handlers.tsx

```diff
@@ -171,7 +171,7 @@ export function getHandlers(platformSDK: LedgerLiveApi): MethodHandler[] {
         {
           name: "message",
           component: JSONTextArea,
-          modifier: (value: any) => value,
+          modifier: (value: any) => Buffer.from(value),
```

Comment:
So this version of the debugger will only work with new version of the platform sdk we need to sync versions between this repo and the LLD/LLM using the `apiVersion` field in the manifest of the debugger